src/components/Header/Logo/index.js

```diff
@@ -0,0 +1,21 @@
+import styled from "styled-components";
+import image from "./img/logo1.png";
+
+function Logo() {
+  return (
+    <LogoWraper>
+      <a href="google.com">
```

Comment:
Well, this is not ok to do so. Remo that href


https://user-images.githubusercontent.com/34886027/208296448-bf3190af-877f-4e24-92c2-652e5fd622c0.mov

Reply:
It's my mistake), I checked it, and I forgot to remove the href.